spyder/plugins/editor/widgets/editor.py

```diff
@@ -1100,6 +1101,13 @@ def set_tabmode_enabled(self, state):
             for finfo in self.data:
                 finfo.editor.set_tab_mode(state)
 
+    def set_stripmode_enabled(self, state):
+        # CONF.get(self.CONF_SECTION, 'strip_automatic_spaces_only')

```

Comment:
This should probably be either uncommented or removed?

Reply:
well every other function has that.

Reply:
@ccordoba12 ?

Reply:
Please leave it for now. That's old stuff that Pierre left and it should be fixed in another PR.